scripts/db_migrator.py

```diff
@@ -1007,6 +1019,8 @@ def version_4_0_3(self):
         This is the latest version for master branch
         """
         log.log_info('Handling version_4_0_3')
+        # Updating DNS nameserver
+        self.migrate_dns_nameserver()
```

Comment:
You updated the existing version, but did not create a new version.

Please add new version handler here as a placeholder. 

Also, when you do that please also update the self.CURRENT_VERSION to the new version.

Reply:
Updated